pkg/apis/kudo/v1beta1/instance_types_test.go

```diff
@@ -74,3 +74,97 @@ func TestGetLastExecutedPlanStatus(t *testing.T) {
 		}
 	}
 }
+
+func TestGetPlanInProgress(t *testing.T) {
+	tests := []struct {
+		name             string
+		planStatus       map[string]PlanStatus
+		expectedPlanName string
+	}{
+		{"no plan ever run", map[string]PlanStatus{"test": {
```

Comment:
nit: could you please use named fields (e.g. `name: "no plan ever run"`) for all test cases? It's getting harder to read for nested structs.